app/templates/account/repositories.hbs

```diff
@@ -128,7 +128,7 @@
         {{/if}}
       {{/component}}
     {{/if}}
-    <div id="administerable-repositories">
+    <div id="administrable-repositories">
```

Comment:
Depending on how you do things, this could be an API change. I'm assuming it isn't.